util/__init__.py

```diff
@@ -6,6 +6,8 @@
 
 # Some settings
 # If you run docker with a sudo, set this envvar to the sudo command: 'sudo'
+from util.dummy_aws_credentials import create_dummy_aws_config
```

Comment:
This should be moved up to line 6 above the comments  about SUDO user option. KEep the comments next to the right piece of code.